packit/api.py

```diff
@@ -856,6 +856,99 @@ def sync_release(
             self.dg.local_project.git_repo.git.reset("--hard", "HEAD")
         return pr
 
+    def get_pr_default_title_and_description(self):
+        """Create a default title and description to be used
+        in a PR for updating upstream from a dist-git push.
+
+        Returns:
+            (title, description):
+            The title str will be the first line in the latest commit.
+            The description str will be all the other lines in the latest commit, if any.
+        """
+        message = self.dg.local_project.git_repo.head.commit.message
+        lines = [line for line in message.split("\n")]
+
+        title = lines[0] or f"({self.dg.local_project.commit_hexsha})"
+        description = "\n".join(lines[1:]).strip()
+
+        return (f"Update upstream to last dist-git commit: {title}", description)
```

Comment:
I know I'm nitpicking :wink:  but the title is the most visible output, and "last" means there will be no more commits, which is not true in most cases.
```suggestion
        return (f"Update upstream to latest dist-git commit: {title}", description)
```

Reply:
You are right, I thought I have already fixed it... but I missed it 😅